src/pal/src/cruntime/misc.cpp

```diff
@@ -245,6 +243,7 @@ PAL_qsort(void *base, size_t nmemb, size_t size,
     PERF_EXIT(qsort);
 }
 
+DLLEXPORT
```

Comment:
We have the DLLEXPORT both in the pal.h / palinternal.h and on the implementations. Is there a reason for that? All the implementation files include palinternal.h which in turn includes pal.h. So it seems it should be sufficient to have the DLLEXPORT in the pal.h / palinternal.h.

Reply:
I removed most of these from implementation files, but then placed some them back because ranlib was producing warnings. There were two or few cases (`bsearch` was one of them) where DLLIMPORT was removed from `pal.h` header because it was conflicting with libstdc++ header in one of the Linux distro, placing it in cpp file satisfied the matrix.